app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -126,7 +126,7 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
 
     requestRows.add(<String, Object>{
       'json': <String, Object>{
-        'ID': 'flutter/flutter/${commit.sha}',
+        'ID': 'flutter/flutter/${commit.branch}/${commit.sha}',
```

Comment:
Is this compatible with all the data that shows up in the current view of the dashboard?

Reply:
This does not affect the dashboard. This is for query purpose in BigQuery only. And dashboard is already using `branch/commit`.